---
title: C28163 | Microsoft Docs
ms.custom: 
ms.date: 11/04/2016
ms.reviewer: 
ms.suite: 
ms.technology: vs-ide-code-analysis
ms.tgt_pltfrm: 
ms.topic: article
f1_keywords: C28163
helpviewer_keywords: C28163
ms.assetid: 24fecbde-1c96-4a45-82f7-9f47cfc0ef11
caps.latest.revision: "3"
author: mikeblome
ms.author: mblome
manager: ghogen
ms.workload: multiple
ms.openlocfilehash: b979514793fbc65fcdf87090065171ec0a4f11c3
ms.sourcegitcommit: 32f1a690fc445f9586d53698fc82c7debd784eeb
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 12/22/2017
---
# <a name="c28163"></a>C28163
Warnung C28163: Die Funktion sollte nie innerhalb eines try/except-Blocks aufgerufen werden.  
  
 Diese Warnung wird ausgegeben, wenn eine Funktion von einem Typ ist nie in eingeschlossen werden soll eine `try/except` Block befindet sich einem `try/except` Block. Das Codeanalysetool gefunden mindestens einen Pfad, in dem die Funktion aufgerufen, in wurde, einem `try/except` Block.